recipes/serapeum_utils/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "serapeum_utils" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Serapieum-of-alex/serapeum_utils/archive/{{ version }}.tar.gz
+  sha256: 657d84d21ffaba7d785d39ab296d7130618c9179d6db5bed831a3728ca379403
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.9,<3.11
+    - pip >=22.3.1
+    - poetry >=1.2.2
+  run:
+    - python  >=3.9,<3.11
+    - pip >=22.3.1
+
+test:
+  imports:
+    - serapeum_utils
+
+about:
+  home: https://github.com/Serapieum-of-alex/serapeum_utils
+  license:  GPL-3.0-only
+  license_family: GPL
+  license_file: LICENSE.md
+  summary: python utility package
```

Comment:
This is quite short. And the description, that should be longer, is the same! Can you add a better summary and description so others finding this package via conda know what it is?

Reply:
I added more description in the description part

Reply:
Is that enough?